### ISSUE TITLE
Malformed OGG files

### ISSUE THREAD

**Issue by xvitaly** [TelegramMessenger/Telegram-iOS#47]
Telegram for iOS produce malformed OGG files for voice messages. It cannot be played on Telegram Desktop built against modern ffmpeg versions (4.1+).

If you will export this OGG file using Telegram Desktop (RMB -> Save As) and then use mpv to play it, you will get an error:

```
[ffmpeg/demuxer] ogg: Packet processing failed: Invalid data found when processing input
[lavf] error reading packet.
[lavf] EOF reached.
```

Telegram Desktop also cannot play it as reported [here](https://github.com/telegramdesktop/tdesktop/issues/5843#issuecomment-495222445):

```
[2019.05.23 14:55:58] Audio Info: recreating audio device and reattaching the tracks
[2019.05.23 14:55:58] Streaming Error: Error in av_read_frame (code: -1094995529, text: Invalid data found when processing input).
[2019.05.23 14:55:59] Audio Info: Closing audio playback device.
```

Bug confirmed in versions: Telegram 5.5.1 (iPhone 6, iOS 12.1.4) and Telegram 5.6.3 (iPhone 7, iOS 12.1.2).

Please fix it.

**Comment by shivanraptor** [comment]
It's Telegram Desktop's OGG export problem, not the App itself.

**Comment by xvitaly** [comment]
> It's Telegram Desktop's OGG export problem, not the App itself.

No. This is definitely not a Telegram Desktop issue, because created by Android app OGG files works fine both in Telegram Desktop and mpv.

**Comment by laktyushin** [comment]
Fixed in 5.10 release.

**Comment by fisher122** [comment]
The same error on the native macOS desktop client
